airflow/providers/google/cloud/hooks/stackdriver.py

```diff
@@ -29,7 +29,7 @@
 from google.protobuf.json_format import MessageToDict, MessageToJson, Parse
 from googleapiclient.errors import HttpError
 
-from airflow import AirflowException
+from airflow.exceptions import AirflowException
```

Comment:
Unrelated change here?

Reply:
Other than this, :+1: to the change.

Reply:
This change was already in the master, but the PR preview has not been updated.